### PR TITLE
Fix luametatex

### PR DESCRIPTION
**Pull request by jakobjakobson13**
- add config-option for luametatex
- adapt post-install command for texlive-source

### CODE REVIEW

File: org.freedesktop.Sdk.Extension.texlive.yml [hunk 1]
```diff
@@ -25,7 +25,16 @@ modules:
         sha512: 5b95a63e77953c540ce1b8e6338d7189785f19333a34c73ae97662d5691968ccaaff9bfd712d2006ebec518a6a6545c00adaf97afd22bfc1fa35ab47c8ce4474
         dest-filename: texlive-texmf.tar.xz
   - name: luametatex
-    buildsystem: cmake
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
```
Comment (hfiguiere) [inline org.freedesktop.Sdk.Extension.texlive.yml]: is there a reason to not put them directly in `bin` ? there is only architecture in the flatpak.

File: org.freedesktop.Sdk.Extension.texlive.yml [hunk 2]
```diff
@@ -25,7 +25,16 @@ modules:
...
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - install -m755 luametatex ${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
```
Comment (hfiguiere) [inline org.freedesktop.Sdk.Extension.texlive.yml]:
```suggestion
      - install -Dm755 luametatex -t ${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
```

File: org.freedesktop.Sdk.Extension.texlive.yml (hunk 1 above)
Comment (hfiguiere) [reply org.freedesktop.Sdk.Extension.texlive.yml]: bit this make it the prefix which is likely not what you want.

Comment (jakobjakobson13) [reply org.freedesktop.Sdk.Extension.texlive.yml]: Sorry, I did not get your comments in time. I tried to reproduce the instructions of https://tug.org/pipermail/tex-live/2023-March/048960.html but it took some time.

**Comment by jakobjakobson13** [comment]
@hfiguiere : Unfortunately I did not see you helpful comments in time. If you have any further comments or suggestions to the pull request, please let me know.

**Review by hfiguiere — APPROVED** [review]

**Pull request merged**